builders/manage/dapp/register-foreign-assets.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Register Foreign Assets
+description: Tanssi appchains benefit from native cross-chain communication, enabling smooth and fast token transfers between the token’s native chain and other chains.
+---
+
+# Register Foreign Assets
+
+## Introduction {: #introduction }
+
+Cross-chain asset transfer is crucial because it enables the seamless movement of digital assets across different appchains, enhancing interoperability, liquidity, and user experience. To enable asset transfers to and from two appchains, first, a bidirectional channel must be open between them. Thanks to the Tanssi dApp, opening a channel is an easy and quick task. Please refer to the [Manage Cross-Chain Communication Channels](/builders/manage/dapp/xcm-channels/){target=\_blank} article to know how to do it.
+
+!!! note
+    Opening a bidirectional communication channel requires approval from both appchains governors.
+
+Once your appchain's communication channels are established, you can register other chain's assets (foreign assets) to start operating. This guide will walk you through the process of registering foreign assets using the [Tanssi dApp](https://apps.tanssi.network){target=\_blank}.
+
+--8<-- 'text/builders/manage/dapp/prerequisites.md'
+
+## Accessing the Foreign Assets Registration Panel {: #accesing-foreign-assets-management-panel }
+
+--8<-- 'text/builders/manage/dapp/cross-chain-config-panel.md'
+
+The panel will show your appchain's cross-chain configuration along with several available actions. Regarding foreign assets, the relevant elements you are presented with are:
+
+1. **Registered Assets panel** - this section will group and present to you all registered assets your appchain already has available
+2. **Assets List** - already registered foreign assets will be displayed in this section, along with their associated information such as asset name, symbol, ID, total supply, and appchain ID where it is native
```

Comment:
I'd maybe rename this as "Registered Assets list" ? but not a big deal either way